torch/testing/_internal/common_device_type.py

```diff
@@ -877,6 +877,20 @@ def skipCUDAIfRocm(fn):
 def skipCUDAIfNotRocm(fn):
     return skipCUDAIf(not TEST_WITH_ROCM, "test doesn't currently work on the CUDA stack")(fn)
 
+# Skips a test for specified CUDA versions, given in the form of a list of [major, minor]s.
+def skipCUDAVersionIn(versions : List[List[int]] = None):
```

Comment:
```suggestion
def skipCUDAIfVersionIn(versions : List[List[int]] = []):
```

Reply:
This runs into error B006 for flake8 as [] is mutable.